File: 1347-minimum-number-of-steps-to-make-two-strings-anagram/1347-minimum-number-of-steps-to-make-two-strings-anagram.cs
public class Solution {
    public int MinSteps(string s, string t) {
        
        int[] a = new int[26];
        for(int i=0;i < s.Length;i++){
            a[s[i]-'a']++;
            a[t[i]-'a']--;
        }
        int c=0;
        foreach(int x in a){
            if(x<0){c+=Math.Abs(x);}
        }
        return c;
    }
}

File: 1347-minimum-number-of-steps-to-make-two-strings-anagram/1347-minimum-number-of-steps-to-make-two-strings-anagram.java
class Solution {
    public int minSteps(String s, String t) {
        int n1 = s.length();
        int[] a= new int[26];
        int[] b= new int[26];
        for(int i=0;i<n1;i++){
            a[s.charAt(i)-'a']++;
            b[s.charAt(i)-'a']--;
            b[t.charAt(i)-'a']++;
        }
        int c=0;
        for(int x: b){
            // if(x>0 ) c++;
             if(x<0) c+= Math.abs(x);
        }
        return c;
    }
}




